gno.land/pkg/integration/testing_integration.go

```diff
@@ -211,10 +230,73 @@ func setupGnolandTestScript(t *testing.T, txtarDir string) testscript.Params {
 
 				tsValidateError(ts, "gnokey", neg, err)
 			},
+			"sleep": func(ts *testscript.TestScript, neg bool, args []string) {
+				d := time.Second
+				if len(args) > 0 {
+					var err error
+					if d, err = time.ParseDuration(args[0]); err != nil {
+						ts.Fatalf("unable to parse duration %q: %s", args[1], err)
+					}
+				}
+
+				time.Sleep(d)
+			},
+			// adduser commands must be executed before starting the node; it errors out otherwise.
+			"adduser": func(ts *testscript.TestScript, neg bool, args []string) {
```

Comment:
What do you think having the possibility to add multiple users at once? like `adduser user1 user2 user3`

Reply:
I prefer not to create multiple at once for a few reasons:
- `addusers` makes is seem like multiple users are being added even if only one is provided
- and I don't think we'd want to support both `adduser` and `addusers`
- when looking at the txtar script, it is generally easier to read this:
```
adduser tester
adduser hello
adduser actor
adduser admin
```
than it is to read this
```
adduser tester hello actor admin
```